src/ContentRepository/Packaging/Steps/EditConnectionString.cs

```diff
@@ -10,6 +10,9 @@ public class EditConnectionString : EditXmlNodes
         public string ConnectionName { get; set; } = "SnCrMsSql";
         public string DataSource { get; set; }
         public string InitialCatalogName { get; set; }
+        public string UserName { get; set; }
+        public string Password { get; set; }
+        public bool IntegratedSecurity { get; set; } = false;

```

Comment:
The point is: we do not need this property, because its value can be calculated from the username (meaning if that is given or not). It could become confusing if you can provide this value separately.